Python's built-in rational in the documentation; integer division in Py 2; tuple behavior (#26786)
sympy/sympy: doc/src/explanation/gotchas.rst

do ``<number>/<number>`` inside of a SymPy expression, Python will
evaluate the two numbers before SymPy has a chance to get
to them.  The solution is to :func:`~.sympify` one of the numbers, or use

> :obj:`~.Rational` (or alternatively Python's `Fraction

Reviewer: ```suggestion
:obj:`~.Rational` (or Python's `Fraction
```

Reviewer 2: While this is true I don't think this is going to be the easiest solution for most people. If you are going to construct a class, you should just construct the SymPy class.

Author: While using `sympy.Rational(fractions.Fraction(x, y))` is an overkill and I don't expect anyone to do so (unless that `Fraction` is a result of another calculation), I think it's good to make the users aware that Python also has a rational class with unlimited precision, not just `float`.

    >>> print(solve(7*x -22, x))
    [22/7]

>     >>> 22/7  # If we just copy and paste we get int 3 or a float

Reviewer: ```suggestion
    >>> 22/7  #copy and paste gives int 3 (in Python 2) or a float (in Python 3)
```

Reviewer: And then the additions to the following line are not needed.

    >>> S("22/7")
    22/7

> Also, if you use Python 2 and do not use :command:`isympy`, you could use ``from

Reviewer 2: We should just remove all mentions of this. No one uses Python 2 any more.

Author: I agree with that, but IMO it deserves a separate PR, though.
Anyway, [the documentation](https://docs.sympy.org/latest/install.html) says:
> SymPy officially supports Python 3.8, 3.9, 3.10, and PyPy.

    >>> (x)
    x

>     Actually parentheses are not needed for non-empty tuples; the commas are:

Reviewer: ```suggestion
    Parentheses are not needed for non-empty tuples; the commas are:
```
